src/transform/ThemeTransform.js

```diff
@@ -74,10 +74,19 @@ const classifyElements = element => {
   /* en > Away colours > 793128975 */
   /* en > Manchester United F.C. > 793244653 */
   /* en > Pantone > 792312384 */
-  Polyfill.querySelectorAll(element, 'div.color_swatch div, div[style*="position: absolute"]')
-    .forEach(div => {
-      div.classList.add(CONSTRAINT.DIV_DO_NOT_APPLY_BASELINE)
-    })
+  /* en > Wikipedia:Graphs_and_charts > 801754530 */
+  /* en > PepsiCo > 799739023 */
+  const selector = [
+    'div.color_swatch div',
+    'div[style*="position: absolute"]',
+    'table.infobox div[role*="img"] div[style*="background-color"]',
```

Comment:
This line will get false positives and should probably be removed. Removing it makes the 'Single statistic' graph not work, but this approach would only work in the case where a single statistic graph happened to be contained by an infobox in the exact way it is on the 'Graphs and charts' page. 

The other query selector items look good though! They're all scoped and more specific.

I confirmed that I don't see any regressions now with the other theme demo articles, either on Dark or Sepia.